tests/typecheck/managers/querysets/test_from_queryset.yml

```
@@ -384,3 +384,37 @@
                 MyManager = BaseManager.from_queryset(MyQuerySet)
                 class MyModel(models.Model):
                     objects = MyManager()
+
+
+-   case: from_queryset_custom_auth_user_model
```

Think we should disable cache for this test also

```suggestion
-   case: from_queryset_custom_auth_user_model
    disable_cache: true
```

---

Are you sure? As this is set up right now it'll run the tests both with and without caching, doesn't it add some value to test that this works _with_ caching as well?

---

I'm not sure there's a consistent way of testing with cache? At least not in a single test suite run.

But I'm quite sure this should run with cache disabled, considering regression etc etc, I'd otherwise, personally, call the test flaky. Since it shouldn't allow the test suite to pass

---

Good point, I've committed your suggestion now :)